sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/generated/LocationComponentSettingsBase.kt

```diff
@@ -64,6 +64,18 @@ abstract class LocationComponentSettingsBase : LocationComponentSettingsInterfac
       applySettings()
     }
 
+  /**
+   * Whether the bearing image of location puck is shown.
+   */
+  override var showBearingImage: Boolean
+    get() {
+      return this.internalSettings.showBearingImage
+    }
+    set(value) {
+      this.internalSettings.showBearingImage = value
```

Comment:
should we check that value changed? I can set twice same value.
maybe call `applySettings` only when it's really changed?

Reply:
All the other properties are the same. I would like to align with them as this is generated.

Reply:
maybe create a ticket?

Reply:
Done.